testing/visual-regression/backstop.json

```diff
@@ -403,6 +403,54 @@
         }
       ]
     },
+    {
+      "label": "3_Components_NAVIGATION_BUTTONS",
+      "cookiePath": "backstop_data/engine_scripts/cookies.json",
+      "url": "http://host.docker.internal:6006/iframe.html?id=3-components--navigation",
+      "referenceUrl": "http://host.docker.internal:6006/iframe.html?id=3-components--navigation",
+      "readyEvent": "",
+      "readySelector": ".js-CadsGreedyNav",
+      "delay": 750,
+      "hideSelectors": [],
+      "removeSelectors": [
+        ".cads-styleguide__breakpoint-label",
+        ".cads-styleguide__language-label",
+        ".cads-styleguide-max-content-width",
+        "h1"
+      ],
+      "keyPressSelectors": [
+        { "selector": ".cads-nav-link", "keyPress": "Tab" }
+      ],
+      "clickSelector": "",
+      "postInteractionWait": 1000,
+      "selectors": [],
+      "selectorExpansion": true,
+      "expect": 0,
+      "misMatchThreshold": 0.1,
+      "requireSameDimensions": true,
+      "viewports": [
+        {
+          "label": "phone",
+          "width": 320,
+          "height": 400
+        },
+        {
+          "label": "Small",
+          "width": 641,
```

Comment:
are these numbers correct? It feels weird that we've gone from 320 x 400 to 641 x 200.

The ones below look weird too. But maybe only the width matters. I'm not particularly clued up on this area, so disregard if not relevant.

Reply:
I coppied these from the other navigation test, I can't comment either way. If I had to guess it's related to the triggering the greedy nav  behaviour.